recipe/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "yara-python" %}
+{% set version = "4.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 7af4354ee0f1561f51fd01771a121d8d385b93bbc6138a25a38ce68aa6801c2c
+  patches:
+    - patches/0001-fix-openssl.patch
+    - patches/0002-fix-enable-openssl.patch
+    - patches/0003-bypass-openssl-and-remove-hardcoded-paths.patch
+    - patches/0004-enable-optional-modules.patch
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make        # [unix]
+    - patch       # [not win]
+    - m2-patch    # [win]
+  host:
+    - python
+    - setuptools
+    - pip
+    - wheel
+    # jansson 2.14 for win (win-64/jansson-2.14-h2bbff1b_0.tar.bz2) 
+    # has a jansson_d.lib and not jansson.lib, nor jansson.dll
```

Comment:
this is fine for this pr, but feels like the jansson package or windows is somehow broken. it is not usual to ship only a debug static library

Reply:
WOuldn't https://github.com/AnacondaRecipes/jansson-feedstock/pull/2 fix the problem @lorepirri ?

Reply:
yes, see dependencies in the description

Reply:
great, that pr looks nice. thanks for taking care!

Reply:
sure! I've updated this one as well, `jansson` is now included with staging channels, and `yara-python` can have `cuckoo` enabled for `win` as well (depends on `jansson`). Thanks for reviewing!